auth-web/src/components/auth/staff/account-management/StaffAccountManagement.vue

```diff
@@ -22,89 +22,35 @@
       </div>
     </header>
     <!-- Tab Navigation -->
-    <v-tabs
-      v-model="tab"
-      background-color="transparent"
-      class="mb-9"
-      @change="tabChange"
-    >
-      <v-tab
-        v-if="canViewAccounts"
-        data-test="active-tab"
-        :to="pagesEnum.STAFF_DASHBOARD_ACTIVE"
+    <template>
+      <v-tabs
+        v-model="tab"
+        background-color="transparent"
+        class="mb-9"
+        @change="tabChange"
       >
-        Active
-      </v-tab>
-
-      <template v-if="canViewInvitations">
-        <v-tab
-          data-test="invitations-tab"
-          :to="pagesEnum.STAFF_DASHBOARD_INVITATIONS"
-        >
-          <v-badge
-            inline
-            color="primary"
-            :content="pendingInvitationsCount"
-            :value="pendingInvitationsCount"
-          >
-            Invitations
-          </v-badge>
-        </v-tab>
-      </template>
-
-      <template v-if="canManageAccounts">
         <v-tab
-          data-test="pending-review-tab"
-          :to="pagesEnum.STAFF_DASHBOARD_REVIEW"
+          v-for="tab in tabs"
+          :key="tab.code"
+          :data-test="tab.code"
+          :to="tab.page"
```

Comment:
Lot of weird artifacts with all the explicit v-tabs along with v-can permission updates. Refactored to use tab config that is built on permissions when loading up and iterates over to render them